modin/_compat/pandas_api/latest/io.py

```diff
@@ -145,6 +153,54 @@ def read_csv(
     return _read(**kwargs)
 
 
+@_inherit_docstrings(pandas.read_html, apilink="pandas.read_html")
+@enable_logging
+def read_html(
+    io,
+    match=".+",
+    flavor=None,
+    header=None,
+    index_col=None,
+    skiprows=None,
+    attrs=None,
+    parse_dates=False,
+    thousands=",",
+    encoding=None,
+    decimal=".",
+    converters=None,
+    na_values=None,
+    keep_default_na=True,
+    displayed_only=True,
+    extract_links=None,
```

Comment:
Do we also need to change `read_html` signature in `modin\core\io\io.py`?

Reply:
maybe, though it's not strictly required as those extra arguments would be handled automatically by `BaseIO.read_html()` accepting `**kwargs` and passing them through, so I would rather not increase the diff